Makefile.am

```diff
@@ -342,3 +342,14 @@ clean-local: clean-docs
 	rm -rf coverage_percent.txt test_bitcoin.coverage/ total.coverage/ fuzz.coverage/ test/tmp/ cache/ $(OSX_APP)
 	rm -rf test/functional/__pycache__ test/functional/test_framework/__pycache__ test/cache share/rpcauth/__pycache__
 	rm -rf osx_volname dist/ dpi36.background.tiff dpi72.background.tiff
+
+test-security:
```

Comment:
Maybe `test-security-check` instead of `test-security`? I mean, it's a test for the binary security check. It doesn't check the security of the actual binaries.

Reply:
Sure. I've updated the target to `test-security-check`.